pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -93,15 +93,11 @@ pub mod pallet {
 		/// The Liquidity Pools message type.
 		type Message: LPEncoding + Clone + Debug + PartialEq + MaxEncodedLen + TypeInfo + FullCodec;
 
-		/// The message router type that is stored for each domain.
-		type Router: DomainRouter<Sender = Self::AccountId>
-			+ Clone
-			+ Debug
-			+ MaxEncodedLen
-			+ TypeInfo
-			+ FullCodec
-			+ EncodeLike
-			+ PartialEq;
+		/// The target of of the messages comming from this chain
```

Comment:
Nit
```suggestion
		/// The target of the messages coming from this chain
```

Reply:
I think Cosmin can fix this if it has not changed in its PR. To avoid conflicts with your PR @cdamian

Reply:
I think I already fixed this.